web/components/feed/feed-items.tsx

```diff
@@ -221,6 +223,27 @@ export function FeedComment(props: {
     money = formatMoney(Math.abs(matchedBet.amount))
   }
 
+  const [highlighted, setHighlighted] = useState(false)
+  const [showToast, setShowToast] = useState(false)
+  const router = useRouter()
+  useEffect(() => {
+    if (router.asPath.includes(`#${comment.id}`)) {
+      setHighlighted(true)
+      setTimeout(() => setHighlighted(false), 3000)
+    }
+  }, [router.asPath])
+
+  function copyLink(event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) {
+    event.preventDefault()
+    let currentLocation = window.location.href
+    if (currentLocation.includes('#')) {
+      currentLocation = currentLocation.split('#')[0]
+    }
+    navigator.clipboard.writeText(`${currentLocation}#${comment.id}`)
```

Comment:
You should use `copyToClipboard` the library function (in copy.ts), which handles weird cases like Firefox on Linux.